src/index.ts

```diff
@@ -7,7 +7,7 @@ import glob from 'fast-glob'
 export interface NetlifyEdgePluginOptions {
   generateStaticManifest?: boolean
   generateEdgeFunctionsManifest?: boolean
-  additionalStaticPaths?: Array<string>
+  additionalStaticPaths?: Array<string> | ((config: ResolvedConfig) => string[])
```

Comment:
[pebble] could we be consistent with `Array<string>` syntax here?

Reply:
Might be worth an eslint rule if this is not a one off. https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/array-type.md

Reply:
Agreed, though in this case it's mostly about consistency